receiver/sqlserverreceiver/factory_others_test.go

```diff
@@ -95,6 +95,8 @@ func TestFactoryOtherOS(t *testing.T) {
 				require.NotEmpty(t, scrapers)
 
 				sqlScrapers = setupSQLServerLogsScrapers(params, cfg)
+				require.Equal(t, 2*sqlScrapers[0].config.ControllerConfig.CollectionInterval.Seconds(), sqlScrapers[0].config.LookbackTime.Seconds(),
```

Comment:
Add the tests for the config in the `config_test`, it is easier to make sure it works there. [See](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f60a7a4b411d8252d340edfaa48703abf61b312d/receiver/sqlserverreceiver/config_test.go#L137).